test/smoke/testApps/FixedRateSampling/src/main/webapp/index.jsp

```diff
@@ -0,0 +1,11 @@
+<%@page pageEncoding="UTF-8" contentType="text/html;charset=UTF-8"%>
+
+<!DOCTYPE html>
+<html>
+<head>
+<title>Simple Telemetry</title>
```

Comment:
Curious why do we name title Simple Telemetry? Shouldn't it reflect the test app name instead? In this case - Fixed Rate Sampling Test App?

Reply:
That was there previously. We can change it, but this page isn't surfaced anywhere except in the code.